[new page] WD-23241 - feat: create solutions/automation page

pedoch opened the pull request:
## Done

- Create solutions/automation page
- Create solutions/automation contact form

## QA

- Check out the [demo](https://canonical-com-1801.demos.haus/solutions/automation) here and compare it to the [design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/canonical.com-solutions?node-id=3998-8602&t=Gxy2ybQeRd5DdSDf-0), and [copy doc](https://docs.google.com/document/d/12KMzjaVbi0ZWpw4BBkBTyOsaexGQdRkwsvW_t3Gw7x4/edit?tab=t.0).
  - Here is the [form copy doc](https://docs.google.com/document/d/174eFlm8V6Oj_ZcSXe4Rc2ODTZMAqN_QqjYFmUolt7qQ/edit?tab=t.0).

## Issue / Card

- [WD-23241](https://warthogs.atlassian.net/browse/WD-23241)


[WD-23241]: https://warthogs.atlassian.net/browse/WD-23241?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

pedoch commented:
@eliman11 I didn't put the previous and next buttons in the tabs because the way tabs are implemented doesn't allow the button to work properly. This might require a lot of work which might be lead to a new vanilla pattern/macro. If it's okay with you we can leave it out. Else I'll try to find a solution.
<img width="1036" height="104" alt="image" src="https://github.com/user-attachments/assets/8824e9c2-f37f-461c-92b5-c313c43f50c5" />

eliman11 commented:
Thanks @pedoch, please address the comments from Lyubo and I can give it another review after. In addition, could you address the following: 
- [ ] Submitting the form seems to take me to this invalid page. Could we have it redirect to the current page (/solutions/automation) instead, with the success banner on top? 
<img width="1728" height="958" alt="Screenshot 2025-07-17 at 12 20 15" src="https://github.com/user-attachments/assets/f085152c-25fd-4b69-ae14-f3c0e195cd96" />

- [ ] Delete additional "of" in "Manage desktop, server and cloud deployments of at enterprise-scale." - annotated on copydoc
- [ ] Missing "Streamline your VM experience with optimized choices, an image library, and a built-in agent for easier operations." as the second bullet point under "Optimize your virtualization layer"
- [ ] "*Source: Enterprise Management Associates report" should link to https://ubuntu.com/blog/enterprise-class-ubuntu-management-with-canonical-landscape as per Matthew's [comment](https://docs.google.com/document/d/12KMzjaVbi0ZWpw4BBkBTyOsaexGQdRkwsvW_t3Gw7x4/edit?disco=AAABnYqUIQE)

- [ ] Quotes seem to be trimming the last line, I think this will be addressed through Lyubo's comment on using the quote pattern
<img width="1426" height="292" alt="Screenshot 2025-07-17 at 15 23 29" src="https://github.com/user-attachments/assets/7c66ebde-1086-4af1-abfd-1dfbfbc8f544" />

- [ ] Logos should have the company name as the alt text
<img width="318" height="727" alt="Screenshot 2025-07-17 at 15 25 44" src="https://github.com/user-attachments/assets/5410dce6-8cb9-4f8a-b348-b23080a02578" />

pedoch commented:
@eliman11 I have addressed your comments as well. Please take another look when you can.

As for the form submission. The URL point to the Canonical domain, since the page doesn't exist yet is show a 404, but the success message shows up it went through successfully. Once the PR is merged this will work as expected.

akbarkz commented:
For some reason content of the tabbed sections disappear on mobile screen:
<img width="371" height="538" alt="Screenshot From 2025-07-18 18-20-01" src="https://github.com/user-attachments/assets/8c26b945-548d-4412-a0f3-07b3c3a55780" />
